<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Research</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>







<link rel="stylesheet" href="styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">GEMMALAB</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="people.html">
    <span class="fa fa-users"></span>
     
    People
  </a>
</li>
<li>
  <a href="research.html">
    <span class="fa fa-cog fa-spin"></span>
     
    Research
  </a>
</li>
<li>
  <a href="publications.html">
    <span class="fa fa-file-text"></span>
     
    Publications
  </a>
</li>
<li>
  <a href="talks.html">
    <span class="fa fa-slideshare"></span>
     
    Photo Album
  </a>
</li>
<li>
  <a href="news.html">
    <span class="fa fa-newspaper-o"></span>
     
    News
  </a>
</li>
<li>
  <a href="collaborators.html">
    <span class="fa fa-handshake-o"></span>
     
    Collaborators
  </a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Research</h1>

</div>


<hr />
<p>We are broadly interested in understanding the metabolic and
morphological mechanisms that plants have developed to adapt to soils
with low nutrient availability. See below the major research interests
of our lab.</p>
<div
id="nsave-maize-n_2o-suppression-and-ammonium-availability-enhancement"
class="section level3">
<h3>üåΩ NSave Maize: <strong>N</strong><span
class="math inline">\(_2\)</span>O <strong>S</strong>uppression and
<strong>A</strong>mmonium a<strong>V</strong>ailability
<strong>E</strong>nhancement</h3>
<p>The University of Illinois, Urbana-Champaign (UIUC), in collaboration
with North Carolina State University, the University of Arizona, and Oak
Ridge National Lab, is developing NSave corn to reduce greenhouse gas
emissions and enhance sustainability. By integrating traits from wild
and modern corn, NSave seeks to lower nitrous oxide emissions by 50%,
decrease fertilizer use by 55%, and sustain high yields. Field trials
and advanced modeling will assess its economic and environmental
impacts, while partnerships with industry aim to promote widespread
adoption and advance climate-resilient agriculture.<br />
<br></p>
</div>
<div
id="nitrogen-use-efficiency-and-phosphorus-use-efficiency-in-maize-and-sorghum"
class="section level3">
<h3>üåΩ Nitrogen Use Efficiency and Phosphorus Use Efficiency in Maize
and Sorghum</h3>
<p>Nitrogen Use Efficiency (NUE) and Phosphorus Use Efficiency (PUE) are
critical measures of how effectively crops utilize nitrogen and
phosphorus, two essential nutrients for plant growth. NUE and PUE refers
to the ability of a plant to produce biomass or grain per unit of N or P
available in the soil respectively. Both are essential for enhancing
crop productivity, particularly in maize and sorghum, which serve as key
staple crops globally. Improving NUE and PUE is crucial not only for
boosting food production to meet the demands of a growing global
population but also for mitigating the environmental consequences of
excessive fertilizer use. High fertilizer application often leads to
nutrient runoff, water pollution, and soil degradation. Our research
focuses on reducing N and P content in maize and sorghum seeds, which
can lead to a decreased reliance on chemical fertilizers. By optimizing
nutrient allocation within the plant, we aim to improve nutrient use
efficiency. Through population genetics analysis and GWAS, we have
identified several alleles and genes that regulate nutrient uptake,
translocation, and utilization.<br />
<br></p>
</div>
<div id="crispr-cas9-mediated-knockout-to-reduce-phosphorus-in-grains"
class="section level3">
<h3>üåΩ CRISPR-Cas9-mediated knockout to reduce phosphorus in grains</h3>
<p>Phosphorus is an essential nutrient for maize productivity, with a
significant portion being allocated to the grains and subsequently
removed during harvest. However, the primary form of phosphorus in maize
grains, phytate, is indigestible by humans, leading to its excretion and
contributing to the eutrophication of aquatic ecosystems. Reducing
phosphorus accumulation in maize grains is therefore critical for
promoting sustainable agricultural practices. This study utilizes
CRISPR-Cas9 gene editing technology to knock out specific genes involved
in phosphorus accumulation, aiming to mitigate its environmental impact
while maintaining maize productivity.<br />
<br></p>
</div>
<div id="exploring-nitrogen-recycling-alleles-from-zea-diploperennis"
class="section level3">
<h3>üåΩ Exploring nitrogen-recycling alleles from <em>Zea
diploperennis</em></h3>
<p>Maize is largely inefficient with nitrogen, leading to significant
contributions to N<span class="math inline">\(_2\)</span>O, a prominent
greenhouse gas. We expect <em>Z. diploperennis</em>, a perennial
teosinte species, to harbor nutrient recycling alleles useful for
improving maize nitrogen use efficiency. To investigate this, we are
assessing lines derived from <em>Z. diploperennis</em> generated as part
of a large population of teosinte introgressed into maize. By planting
them in nitrogen sufficient and deficient conditions, we can use
hyperspectral reflectance, a reliable, nondestructive method of
evaluating physiological traits, to evaluate these lines for nitrogen
content. We hope to discover interesting <em>Z. diploperennis</em>
regions that confer a notable change in N content.<br />
<br></p>
</div>
<div
id="transcriptomic-and-phenotypic-analysis-of-highland-maize-alleles-to-improve-cold-tolerance"
class="section level3">
<h3>üåΩ Transcriptomic and phenotypic analysis of highland maize alleles
to improve cold tolerance</h3>
<p>Early maize planting can improve nitrogen uptake and reduce
emissions, but it also increases exposure to cold temperatures, which
limits growth and yield. Highland maize varieties from Mexico, like
Palomero Toluque√±o, contain unique alleles that have been lost during
maize domestication. These alleles may be key to improving cold
tolerance in modern maize. To investigate this, we are analyzing
recombinant inbred lines (RILs) derived from a cross between Palomero
Toluque√±o and a commercial maize line. Through transcriptomic analysis
and phenotypic evaluations of cold stress responses, we aim to uncover
alleles and pathways that could enhance maize cold resilience, enabling
earlier planting and better adaptation to fluctuating climates.</p>
<div style="text-align: center;">
<p><img src="images/plot.avif" style="width: 300px;"></p>
</div>
<p><br></p>
</div>
<div id="structural-characterization-of-hpc1-in-highland-maize"
class="section level3">
<h3>üåΩ Structural characterization of HPC1 in highland maize</h3>
<p>Previously, we have characterized the HPC1 gene, or High
PhosphatidylCholine 1, and mapped its location within the maize genome.
This gene was introgressed from teosinte mexicana and encodes for
phospholipase A1, an enzyme that cleaves phosphatidylcholine, PC, to
lysophosphatidylcholine, LPC. PC is capable of packing more tightly
within the membrane, whereas LPC has a more fluid structure. In colder
conditions, high levels of LPC in the cell membrane will result in
cellular damage; this is something that can be combated with higher
quantities of PC which is better suited for a stable membrane bilayer.
Highland maize were found to have a SNP within the coding region of HPC1
that mutates the flap lid and prevents functionality in the enzyme
product. This contains a catalytic triad that is covered by a flap lid
that facilitates substrate binding. Although the structure of
phospholipase A1 has been predicted, it has yet to be crystalized to
verify structural predictions regarding enzyme functionality. Finding a
means to manipulate or select for these important structural components
will help to create more adaptive maize.<br />
<br></p>
</div>
<div
id="testing-adaptive-value-of-highland-teosinte-introgression-into-maize"
class="section level3">
<h3>üåΩ Testing adaptive value of highland teosinte introgression into
maize</h3>
<p>This research employs genetic analysis to understand the impact of
highland teosinte introgression in maize local adaptation. Through
genome scans of Mexican traditional varieties and linkage mapping in
controlled crosses, I investigated two key regions from teosinte
mexicana: the phospholipase gene HPC1 and the chromosomal inversion
Inv4m. These loci exhibit strong genotype-by-environment interaction
affecting flowering time. Mechanistic studies suggest that HPC1
modulates flowering through phospholipid metabolism, while Inv4m might
accelerate flowering via a gene network controlling cell proliferation
and shoot apical meristem transition, showing how teosinte mexicana
alleles could enhance stress tolerance and reproductive fitness.</p>
<div style="text-align: center;">
<p><img src="images/manhattan.png" style="width: 600px;"></p>
</div>
<p><br></p>
</div>
<div id="harnessing-lipidomics-for-enhanced-sorghum-stress-tolerance"
class="section level3">
<h3>üåΩ Harnessing Lipidomics for Enhanced Sorghum Stress Tolerance</h3>
<p>Our research leverages lipidomics to find the most important
molecular mechanisms that enhance sorghum‚Äôs resilience to multiple
stress conditions. Using a 400-genotype Sorghum Association Panel, we
conducted a comprehensive lipidomics analysis across control, and low
phosphorus, low nitrogen, and cold stress environment which we define as
low-input environment. By analyzing the lipid profiles under each
condition, we identified stress-responsive lipids and performed
Genome-Wide Association Studies for each lipid, enabling us to pinpoint
genetic loci linked to lipid-mediated stress adaptation. These candidate
genes, now under validation, are being tested for their role in
multi-stress tolerance through functional assays and molecular
analysis.<br />
<br></p>
</div>
<div
id="predictive-modeling-for-adaptive-genotype-selection-and-cross-species-knowledge-transfer"
class="section level3">
<h3>üåΩ Predictive Modeling for Adaptive Genotype Selection and
Cross-Species Knowledge Transfer</h3>
<p>We employ machine learning approaches to predict genotype-environment
interactions and enhance selective breeding strategies for maize and
sorghum. By integrating high-resolution genomic data from landraces with
environmental variables like nitrogen and phosphorus in soil, we aim to
model the adaptive potential of untested genotypes. This enables us to
computationally infer the optimal nitrogen (N) and phosphorus (P)
conditions for diverse genotypic backgrounds, allowing for precision
selection of mapping populations for field validation. We are also
working on deep learning architectures, including transfer learning
frameworks, to bridge metabolomics and lipidomics knowledge across
species specifically maize and sorghum. By training predictive models on
one species and fine-tuning them for another, we exploit shared
biochemical and regulatory patterns, unlocking scalable insights into
stress adaptation and metabolic plasticity.<br />
<br></p>
</div>
<div
id="bzea-a-diverse-teosinte-introgression-population-for-discovering-agronomically-relevant-alleles-in-maize"
class="section level3">
<h3>üåΩ BZea: A diverse teosinte introgression population for discovering
agronomically relevant alleles in maize</h3>
<p>Teosinte, the wild ancestor of maize, harbors a rich reservoir of
interesting traits that have been largely diminished during maize
domestication. Recognizing the potential of these unknown alleles, we
have developed a large teosinte introgression population that consists
of 81 georeferenced teosinte accessions crossed into the temperate
inbred line B73. Evaluating the impact of teosinte alleles on agronomic
traits is inherently difficult due to the differences in photoperiod and
growth habitat of maize. This population, ‚ÄúBZea,‚Äù addresses this
challenge by creating BC<span class="math inline">\(_2\)</span>S<span
class="math inline">\(_3\)</span>s with B73, creating derived lines with
12.5% of the original teosinte donors. Introgression into B73 allows for
the evaluation of teosinte alleles in a maize background in temperate
conditions.<br />
<br></p>
</div>
<div
id="genetic-and-phenotypic-characterization-of-wild-teosinte-alleles-introgressed-into-elite-maize-line-cml311"
class="section level3">
<h3>üåΩ Genetic and phenotypic characterization of wild teosinte alleles
introgressed into elite maize line CML311</h3>
<p>This research focuses on the genetic and phenotypic characterization
of wild teosinte alleles introgressed into the elite maize line CML311.
Using 2,208 near-isogenic lines (NILs) derived from crosses between 85
teosinte plants (from 78 wild populations spanning nearly all Zea taxa)
and the tropical maize line CML311, the goal is to investigate the
potential of the wild alleles to improve modern maize. Introgressions
were mapped using sequencing techniques, and phenotypic evaluations in
three Mexican environments analyzed the consistency of QTLs among
teosinte donors.<br />
<br></p>
</div>
<div id="genetics-of-glycerolipid-metabolism-in-highland-maize"
class="section level3">
<h3>üåΩ Genetics of glycerolipid metabolism in highland maize</h3>
<p>Maize (<em>Zea mays</em>) originated from Balsas teosinte (<em>Zea
mays ssp. parviglumis</em>) in southwest Mexico around 9,000 years ago,
with a secondary subspecies, Chalco teosinte (<em>Zea mays ssp.
mexicana</em>), found in the cooler, drier highlands of central Mexico.
After domestication, maize spread to the Mexican highlands, potentially
benefiting from introgression with mexicana. Maize later migrated to
South America, where a second highland colonization event occurred in
the Andes.</p>
<p>In both Mesoamerican and South American highlands, maize adapted to
harsh environmental conditions, including higher radiation, lower air
pressure, and temperature, as well as volcanic Andosol soils with low
phosphorus bioavailability. This colonization serves as an ideal model
for studying plant local adaptation and convergence/divergence in
similar environments.</p>
<p>Maize adapted to phosphorus-limiting soils through metabolic
strategies, such as recycling phosphorus-containing compounds like RNA
and phospholipids. Phospholipids, especially phosphatidylcholine, are
crucial for membrane structure and function, including responses to
environmental stimuli. In low-temperature conditions, plants increase
phospholipids in membranes to modulate fluidity, while under phosphorus
deficiency, they substitute phospholipids with galactolipids and
sulfolipids. We hypothesize that maize‚Äôs glycerolipid metabolism
underwent strong selective pressure during its adaptation to the Mexican
highlands.</p>
<p>Through quantitative genetics analysis of glycerolipid metabolism in
maize from various mapping populations grown in highland and lowland
conditions, we aim to identify loci involved in lipid remodeling in
highland maize.<br />
<br></p>
</div>
<div
id="improving-candidate-gene-discovery-by-combining-multiple-genetic-mapping-datasets"
class="section level3">
<h3>üåΩ Improving candidate gene discovery by combining multiple genetic
mapping datasets</h3>
<p>Identifying genes involved in plant adaptation to local habitats and
environmental stresses is a key goal for plant scientists, aiding in
crop breeding and understanding plant development, evolution, and stress
responses. However, validating gene function and determining the impact
of different gene variants is complex, especially when multiple genes
contribute to a trait. This project aims to investigate the effects of
phosphorus deficiency and cold stress on sorghum lipid metabolism and
develop mathematical approaches to integrate data from genome-wide
association studies and population genetics. By combining metabolic
profiling of sorghum lines under stress with genetic differentiation
from geolocated natural populations, the project seeks to identify and
prioritize candidate genes and metabolic pathways. These methods can be
applied to bioenergy crops and other environmental stresses.</p>
<p><br> Have questions or want to know more? Reach out to <a
href="mailto:rrellan@ncsu.edu">Dr.¬†Rell√°n-√Ålvarez</a> and use
‚ÄòSupercalifragilisticexpialidocious‚Äô as the code word in the subject
line!<br />
<br></p>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
