<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Talks</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/font-awesome-6.4.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.2/css/v4-shims.min.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>







<link rel="stylesheet" href="styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">GEMMALAB</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="people.html">
    <span class="fa fa-users"></span>
     
    People
  </a>
</li>
<li>
  <a href="research.html">
    <span class="fa fa-cog fa-spin"></span>
     
    Research
  </a>
</li>
<li>
  <a href="publications.html">
    <span class="fa fa-file-text"></span>
     
    Publications
  </a>
</li>
<li>
  <a href="photos.html">
    <span class="fa fa-camera"></span>
     
    Photo Album
  </a>
</li>
<li>
  <a href="news.html">
    <span class="fa fa-newspaper-o"></span>
     
    News
  </a>
</li>
<li>
  <a href="collaborators.html">
    <span class="fa fa-handshake-o"></span>
     
    Collaborators
  </a>
</li>
<li>
  <a href="talks.html">
    <span class="fa fa-slideshare"></span>
     
    Talks
  </a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Talks</h1>

</div>


<hr />
<p><strong>Conference talks and invited seminars</strong></p>
<p><strong>2025</strong></p>
<p><strong>53</strong>. Mohammed VI Polytechnic University, Morocco, IV
Science Week, 2025, <strong>Invited Talk</strong>. “Using crop natural
genetic variation to improve phosphorus efficiency”</p>
<p><strong>2024</strong></p>
<p><strong>52</strong> North Carolina State University, 2024, Intrinsyc
Seminar Series. Invited Seminar. “CERCA: Circular Economy that
Reimagines Corn Agriculture”</p>
<p><strong>2023</strong></p>
<p><strong>51</strong>. China Agricultural University, Workshop on Plant
Responses to Abiotic Stresses and Environmental Signals 2023,
<strong>Invited Talk</strong>. “North, South, Up and Down. The genes and
molecules that allow maize to move all around.”</p>
<p><strong>50</strong>. University of California Riverside 2023,
<strong>Invited Seminar</strong>. “North, South, Up and Down. The genes
and molecules that allow maize to move all around.”</p>
<p><strong>49</strong>. Plant Breeding Consortium, North Carolina State
University. International Plant Breeding Consortium, <strong>Invited
Seminar</strong>. “The genetics of Maize Wild Relatives and Maize Native
Varieties local adaptation”</p>
<p><strong>48</strong>. University of Madison Wisconsin 2023,
<strong>Invited Seminar</strong>. “North, South, Up and Down. The genes
and molecules that allow maize to move all around.”</p>
<p><strong>2022</strong></p>
<p><strong>47</strong>. University of Nebraska Lincoln. Center for
Biological Chemistry and Redox Biology Center. 2022, <strong>Invited
Seminar</strong>. “North, South, Up and Down. The genes and molecules
that allow maize to move all around.”</p>
<p><strong>2021</strong></p>
<p><strong>46</strong>. Wellcome Trust Plant Genomes in a Changing
Environment Conference 2021, <strong>Invited Talk</strong>. John Innes
Center, UK. “The molecular and genetic basis of maize landrace local
adaptation”</p>
<p><strong>45</strong>. Maize Genetics Meeting 2021 <strong>Invited
Talk</strong>. “Teosinte introgression modulates phosphatidylcholine
levels and induces early maize flowering time.”</p>
<p><strong>44</strong>. Genetic Engineering and Society Center
Colloquium. <strong>Invited Seminar</strong>. “Drinking from the maize
diversity firehose.”</p>
<p><strong>42</strong>. Department of Plant Sciences, Pennsylvania State
University. Invited Seminar. Invited Seminar. “North, South, Up and
Down. The genes and molecules that allow maize to move all around”.</p>
<p><strong>2020</strong></p>
<p><strong>41</strong>. Genetics and Genomics Seminar Series, North
Carolina State University. Invited Seminar. “North, South, Up and Down.
The genes and molecules that allow maize to move all around”.</p>
<p><strong>40</strong>. Plant Sciences &amp; Plant Pathology Department,
Montana State University. Invited Seminar“North, South, Up and Down. The
genes and molecules that allow maize to move all around.”</p>
<p><strong>2019</strong></p>
<p><strong>39</strong>. American Seed Trade Association 2019, Chicago.
Invited Talk. “The genetics of phospholipid metabolism in highland
maize.”</p>
<p><strong>38</strong>. Plant and Microbial Biology Department,
University of Minnesota. Invited Seminar. “Evolutionary lipidomics of
maize local adaptation”</p>
<p><strong>37</strong>. North Carolina Plant Molecular Biology retreat,
<strong>Invited talk</strong>.</p>
<p><strong>36</strong>. Plant and Molecular Biology department, NC
State, <strong>Invited talk</strong>.</p>
<p><strong>35</strong>. La Molina University in Lima, Perú.
<strong>Invited Talk</strong>. “Evolutionary lipidomics of maize local
adaptation”</p>
<p><strong>34</strong>. Germplasm Enhancement of Maize meeting in
Clayton, NC, <strong>Invited Talk</strong></p>
<p><strong>33</strong>. Syngenta Plant Stress Symposium at Syngenta
headquarters in the Research Triangle Park, NC. <strong>Invited
Talk</strong></p>
<p><strong>32</strong>. Plant Biology Department, UGA in Athens, Georgia
<strong>Invited Seminar</strong></p>
<p>2018</p>
<p><strong>31</strong>. Misión Biológica de Galicia in Pontevedra,
Spain. <strong>Invited Seminar</strong></p>
<p><strong>30</strong>. Aula Dei Experimental Station in Zaragoza,
Spain. <strong>Invited Seminar</strong></p>
<p><strong>29</strong>. MexPopGen meeting. Irapuato, Mexico.
<strong>Invited talk</strong></p>
<p><strong>2017</strong></p>
<p><strong>28</strong> Iowa State University. Genetics and Genomics
Graduate Group. <strong>Invited Seminar</strong>. <a
href="http://bit.ly/rra_ISU_2017">Towards an understanding of maize
adaptation to highlands using a combination of lipidomics and
quantitative and population genetics</a>.</p>
<p><strong>27</strong> Donald Danforth Plant Science Center Scientific
Retreat. Mo. <strong>Invited Plenary Seminar.</strong> <a
href="https://speakerdeck.com/rellan/convergent-glycerolipid-metabolism-in-highland-adapted-maize">Convergent
phospholipid metabolism in highland adapted maize?</a></p>
<p><strong>26</strong> Michigan State University. <strong>Invited
seminar</strong>, Laboratory of David Lowry. Identification of
phospholipid metabolic patterns involved in maize adaptation to low
phosphorus and cold conditions.</p>
<p><strong>25</strong> UC Davis, Genome Center. <strong>Invited</strong>
<strong>Seminar</strong>. Identification of phospholipid metabolic
patterns involved in maize adaptation to highland conditions.</p>
<p><strong>24</strong> Carnegie Institution for Science. Plant Biology
Department, Stanford, CA. <strong>Invited</strong>
<strong>Seminar</strong>. Identification of phospholipid metabolic
patterns involved in maize adaptation to highland conditions.</p>
<p><strong>23</strong> Gordon Research Conference Plant Lipids.
Structure, Metabolism &amp; Function, Galveston, TX. <strong>Invited
talk</strong>. The Role of Phospholipid Metabolism in Maize Adaptation
to Low Phosphorus and Cold Conditions.</p>
<p><strong>22</strong> Future Trends in Common Bean and Maize Research.
Instituto de Biotecnología, Cuernavaca, México. <strong>Invited
talk</strong>. Convergent phospholipid metabolism in highland adapted
maize?</p>
<p><strong>2016</strong></p>
<p><strong>21</strong> 7th International Crop Science Congress. Beijing,
China. <strong>Invited talk.</strong> <a
href="https://speakerdeck.com/rellan/the-role-of-phospholipid-metabolism-in-maize-adaptation-to-low-phosphorus-and-cold-conditions">The
role of polar lipid metabolism reorganization in maize adaptation to low
phosphorus soils of the Trans-Mexican volcanic belt highlands</a>.</p>
<p><strong>20</strong> Langebio Academic Days, Langebio, Mexico.
<strong>Internal Seminar</strong>. <a
href="https://speakerdeck.com/rellan/towards-a-faster-more-open-and-collaborative-science-the-future-of-scientific-publishing-beyond-journals-and-impact-factors">Towards
a faster, more open, and more collaborative science. The future of
scientific publishing beyond journals and impact factors</a></p>
<p><strong>19</strong> Instituto de Ecología, UNAM, México City, México.
<strong>Invited seminar</strong> <a
href="https://speakerdeck.com/rellan/multidimensional-mapping-of-root-responses-to-soil-environmental-cues-using-a-luminescence-based-imaging-system">Multidimensional
mapping of root responses to soil environmental cues using a
luminescence-based imaging system</a></p>
<p><strong>18</strong> Universidad de Guanajuato, Guanajuato, México.
<strong>Invited seminar</strong> <a
href="https://speakerdeck.com/rellan/multidimensional-mapping-of-root-responses-to-soil-environmental-cues-using-a-luminescence-based-imaging-system">Multidimensional
mapping of root responses to soil environmental cues using a
luminescence-based imaging system</a></p>
<p><strong>2015</strong></p>
<p><strong>17</strong> XVI National Congress of Biochemistry and Plant
Molecular Biology, IX Simposium Mexico-USA Queretaro, México.
<strong>Keynote talk</strong>. Towards a root system-level understanding
of how plants adjust root function and shape and integrate heterogeneous
environmental cues.</p>
<p><strong>16</strong> Leibniz Institute of Plant Genetics and Crop
Plant Research. Gatersleben, Germany. <strong>Invited seminar</strong>
<a
href="https://speakerdeck.com/rellan/multidimensional-mapping-of-root-responses-to-soil-environmental-cues-using-a-luminescence-based-imaging-system">Multidimensional
mapping of root responses to soil environmental cues using a
luminescence-based imaging system</a></p>
<p><strong>15</strong> UMR “Biochemistry and Plant Molecular Physiology”
- INRA Montpellier, France. <strong>Invited seminar</strong> <a
href="https://speakerdeck.com/rellan/multidimensional-mapping-of-root-responses-to-soil-environmental-cues-using-a-luminescence-based-imaging-system">Multidimensional
mapping of root responses to soil environmental cues using a
luminescence-based imaging system</a></p>
<p><strong>14</strong> Rhizosphere 4, Maastricht, The Netherlands.
<strong>Roundtable Organization</strong> Emerging technologies for root
systems scale imaging and phenotyping.</p>
<p><strong>13</strong> Instituto de Biotecnología, UNAM, Cuernavaca,
México. <strong>Invited seminar</strong> <a
href="https://speakerdeck.com/rellan/multidimensional-mapping-of-root-responses-to-soil-environmental-cues-using-a-luminescence-based-imaging-system">Multidimensional
mapping of root responses to soil environmental cues using a
luminescence-based imaging system</a></p>
<p><strong>2014</strong></p>
<p><strong>12</strong> BASF 2014 Symposium on Unlocking Yield Potential
in Soil. Limburgerhof, Germany. <strong>Invited talk</strong></p>
<p><strong>11</strong> Annual Scientific Meeting of the American Society
of Plant Biology. Portland, USA. Understanding Root Physiology in Soil
Using a Novel Imaging Platform. <strong>Selected talk</strong></p>
<p><strong>10</strong> PAG XXII Plant Phenotypes Workshop. San Diego,
USA. <strong>Selected talk.</strong> Growth and Luminescence Observatory
of Roots (GLO-Roots) A platform for the Analysis of Root Structure and
Physiology in Soil</p>
<p><strong>09</strong> San José University, CA. <strong>Invited
seminar</strong> Understanding Root Physiology in Soil Using a Novel
Imaging Platform.</p>
<p><strong>2013</strong></p>
<p><strong>08</strong> XVII International Plant Nutrition Colloquium,
Istanbul, Turkey. <strong>Invited talk. (Marschner Young Scientist
Award)</strong>. Growth and Luminescence Observatory of Roots
(GLO-Roots) A platform for the Analysis of Root Structure and Physiology
in Soil.</p>
<p><strong>07</strong> 30th Annual IPG Symposium on Root Biology,
Columbia, Missouri. <strong>Selected talk</strong>. Growth and
Luminescence Observatory of Roots (GLO-Roots) A platform for the
Analysis of Root Structure and Physiology in Soil.</p>
<p><strong>06</strong> California State University at Monterey Bay, CA.
<strong>Invited seminar</strong>. Understanding Root Physiology in Soil
Using a Novel Imaging Platform.</p>
<p><strong>2012–2009</strong></p>
<p><strong>05</strong> 3rd Japan-China Joint Workshop on Plant
Nutrition, Kurashiki, Japan. (2011). Iron xylem transport, the long and
short of it. <strong>Invited seminar</strong></p>
<p><strong>04</strong> XV International Symposium on Iron Nutrition and
Interactions in Plants, Budapest, Hungary (2010). Delving into iron
deficiency metabolomics. <strong>Selected talk</strong></p>
<p><strong>03</strong> 6th International Franco-Spanish Workshop on
Bio-Inorganic Analytical Chemistry, Pau, Francia. 2010 Iron speciation
in plant xylem sap using LC-ESI-TOFMS. <strong>Selected
talk</strong></p>
<p><strong>02</strong> XVI International Plant Nutrition Colloquium.
Sacramento, California, USA (2009). Changes in the proteomic and
metabolic profiles of Beta vulgaris root tips in response to iron
deficiency and resupply <strong>Selected talk</strong></p>
<p><strong>01</strong>. XVIII Reunión de la Sociedad Española de
Fisiología Vegetal. XI Congreso Hispano-Luso de Fisiología Vegetal,
Zaragoza, Spain (2009). Iron is transported as a tri-Fe(III),
tri-citrate complex in plant xylem sap. <strong>Selected
talk</strong></p>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
